imagetest/utils/test_utils.go

```diff
@@ -238,3 +242,83 @@ func GetInterface(index int) (net.Interface, error) {
 
 	return GetInterfaceByMAC(mac)
 }
+
+// WindowsOnly skips tests not on Windows.
+func WindowsOnly(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		t.Skip("Test only run on Windows.")
+	}
+}
+
+// Skip32BitWindows skips tests on 32-bit client images.
+func Skip32BitWindows(t *testing.T, skipMsg string) {
+	image, err := GetMetadata("image")
+	if err != nil {
+		t.Fatalf("couldn't get image from metadata")
+	}
+
+	if strings.Contains(image, "-x86") {
+		t.Skip(skipMsg)
+	}
+}
+
+// PowershellOutput holds stdout, stderr and the exit code from running a powershell command.
+type PowershellOutput struct {
```

Comment:
ProcessStatus would sound better than PoweshellOutput and would embrace future uses avoiding possible duplications